src/utils/url.ts

```diff
@@ -91,6 +99,6 @@ export const urlUtils = {
   getUrlToPath,
   getUrlWithCompanyNumber,
   getUrlWithCompanyNumberTransactionIdAndSubmissionId,
-  sanitiseReqlUrls,
+  sanitiseReqlUrls: sanitiseReqUrls,
```

Comment:
Rather than do this and keep supporting the misspelt function name I would just have sanitiseReUrls and update all the places where the old function name was used.